docs/design-language.md

```diff
@@ -223,4 +223,33 @@ Panels come with an assortment of configurable navigation controls:
 
 ![panels](/img/panels/panel-welcome.jpg "Welcome Dialog Panel") ![panels](/img/panels/panel-feature.jpg "Features Panel")
 
+## Dialogs
+
+### Modal Dialogs
+
+Occasionally, there will be a need to display information, warn the user, or present controls in a manner that needs to utilize a more focused interface to the user.
```

Comment:
1. I think we should talk about how modal dialogs can be used with our flexible layout strategy to put any type of content into a modal dialog. And the content that best fits a modal is the type that you want to user to focus on without interacting with the rest of the app.

1. Warning the user falls more under the alert category than the modal sections.

Reply:
Updated the wordage.

Reply:
@Woolfe-Glitch, I don't see any updated wording here.

Reply:
Information about "warning the user" is better suited for the "Alerts" section.

Reply:
Are you going to add in a section about "Notifications"?

Reply:
Notification section has been added. Thank you.